samples/layouts/dock-manager/split-pane-fixed-size/src/index.ts

```diff
@@ -0,0 +1,83 @@
+import './DockManagerStyles.css'
+import { defineCustomElements } from 'igniteui-dockmanager/loader';
+import { 
+    IgcDockManagerPaneType,
+    IgcSplitPaneOrientation,
+    IgcDockManagerComponent
+} from 'igniteui-dockmanager';
+
+defineCustomElements();
+
+export class DockManagerSplitPaneFixedSize {
+    private dockManager: IgcDockManagerComponent;
+
+    constructor() {
+        this.dockManager = document.getElementById('dockManager') as IgcDockManagerComponent;
+        this.dockManager.dir = 'ltr';
+
+        this.dockManager.layout = {
+            rootPane: {
+                type: IgcDockManagerPaneType.splitPane,
+                orientation: IgcSplitPaneOrientation.vertical,
```

Comment:
How about we make the root useFixedSize: true as well?
```suggestion
                orientation: IgcSplitPaneOrientation.vertical,
                useFixedSize: true,
```